pkglistgen/tool.py

```diff
@@ -912,7 +915,7 @@ def update_and_solve_target(
         if git_url:
             if product_dir and os.path.isdir(product_dir):
```

Comment:
if the two modes are exclusive, I think this if block can be dropped as well.

Reply:
thank you, agreed